.rubocop_todo.yml

```diff
@@ -124,12 +124,6 @@ Rails/Blank:
 Rails/Date:
   Enabled: false
 
-# Offense count: 1
-# Cop supports --auto-correct.
-Rails/Delegate:
```

Comment:
This should be in the first commit. Only the addition of the link should be here. :wink: